docs/reference/gps.md

```diff
@@ -169,9 +169,9 @@ The returned vector indicates the absolute position of the [GPS](#gps) device.
 This position can either be expressed in the cartesian coordinate system of Webots or using latitude-longitude-altitude, depending on the value of the `gpsCoordinateSystem` field of the [WorldInfo](worldinfo.md) node.
 The `gpsReference` field of the [WorldInfo](worldinfo.md) node can be used to define the reference point of the GPS.
 
-The `wb_gps_get_speed` function returns the current [GPS](#gps) speed in meters per second.
+The `wb_gps_get_speed` function returns the current [GPS](#gps) speed in meters per second. If there is no physic node on the robot, the first returned value will be NAN.
```

Comment:
Here are some changes to make it more consistent with the existing documentation
```suggestion
The `wb_gps_get_speed` function returns the current [GPS](#gps) speed in meters per second.
If there is no physics node on the robot, the first returned value will be *NaN*.
```